src/components/Comment/CommentThread/CommentThread.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import { CommentData } from 'src/types/domain';
+
+import Comment from '../Comment';
+import { CommentThreadWrapper } from './styled';
+
+type CommentThreadProps = {
+  comments?: Array<CommentData>;
+  onReply: (commentData: CommentData) => void;
+  isTopLevel?: boolean;
+};
+
+function CommentThread({
+  comments,
+  onReply,
+  isTopLevel = true,
+}: CommentThreadProps) {
+  if (!comments) {
+    return <></>;
```

Comment:
It's also possible to return `null` in a component to render nothing 😄 It's also more idiomatic React and meaningful since it states to the reader that "this is no mistake, we want to show nothing here".

Reply:
Agreed! Thanks for the feedback, the code became much more clean now!